src/gui/programupdater.cpp

```diff
@@ -36,66 +37,90 @@
 #include <QDebug>
 #include <QDesktopServices>
 #include <QRegularExpression>
-#include <QStringList>
 #include <QXmlStreamReader>
 
 #if defined(Q_OS_WIN)
 #include <QSysInfo>
 #endif
 
 #include "base/net/downloadmanager.h"
+#include "base/utils/version.h"
 #include "base/version.h"
 
 namespace
 {
-    const QString RSS_URL {QStringLiteral("https://www.fosshub.com/feed/5b8793a7f9ee5a5c3e97a3b2.xml")};
+    bool isVersionMoreRecent(const QString &remoteVersion)
+    {
+        using Version = Utils::Version<int, 4, 3>;
 
-    QString getStringValue(QXmlStreamReader &xml);
+        try
+        {
+            const Version newVersion {remoteVersion};
```

Comment:
>Wouldn't it be useful to distinguish the case when there is no more recent version available and the case when there is no valid version string obtained? 

I'm not sure, currently knowing if the remote version is more up-to-date is sufficient and an invalid version string is treated as a "no" answer.

>If so, I would take Utils::Version<int, 4, 3> as a parameter so that the function compares obviously valid versions.

The `<4, 3>` part is matching the format of qbt version, see base/version.h.in. I don't think it can be changed.

>Maybe parse it in ProgramUpdater?

I think it is a bit out of scope of this PR (the changes in this PR preserves the same outcome expectations, more or less) I don't mind someone do it in another PR.